bigtable/api/QuickStart/QuickStart.cs

```diff
@@ -0,0 +1,56 @@
+﻿// [START bigtable_quickstart]
+using System;
+// Imports the Google Cloud client library
+using Google.Cloud.Bigtable.V2;
+
+namespace GoogleCloudSamples.Bigtable
+{
+    class QuickStart
+    {
+        static int Main(string[] args)
+        {
+            // Your Google Cloud Platform project ID
+            const string projectId = "YOUR-PROJECT-ID";
+            // The name of the Cloud Bigtable instance
+            const string instanceId = "YOUR-INSTANCE-ID";
+            // [END bigtable_quickstart]
+            if (projectId == "YOUR-PROJECT" + "-ID")
+            {
+                Console.WriteLine("Edit QuickStart.cs and replace YOUR-PROJECT-ID with your project id.");
+                return -1;
+            }
+            if (instanceId == "YOUR-INSTANCE" + "-ID")
+            {
+                Console.WriteLine("Edit QuickStart.cs and replace YOUR-INSTANCE-ID with your instance id.");
+                return -1;
+            }
+            // [START bigtable_quickstart]
+            // The name of the Cloud Bigtable table
+            const string tableId = "my-table";
+      
+            try
+            {
+                // Creates a Bigtable client
+                BigtableClient bigtableClient = BigtableClient.Create();
+
+                // Read a row from my-table using a row key
+                Row row = bigtableClient.ReadRow(new TableName(projectId, instanceId, tableId), "r1", RowFilters.CellsPerRowLimit(1));
```

Comment:
Please split this over two lines for better readability

Reply:
done